woocommerce/woocommerce
# Update the `Customer Account` block icon for the line style

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:








### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:


1. Create a new page or post.
2. Insert the `Customer Account` block.
3. Check there are 3 styles of icons to choose from on the sidebar settings and choose the first one.
4. Check the block in the editor updates accordingly.
6. Save and check it displays as well in the front end.
7. Check the line for the head and the body of the icon are the same width:

| Before | After |
|--------|--------|
| <img width="145" alt="Screenshot 2024-07-11 at 16 12 19" src="https://github.com/woocommerce/woocommerce/assets/186112/6f9d11b3-5f4f-4752-a43a-4eb6d3e9a034"> | <img width="150" alt="Screenshot 2024-07-11 at 16 12 07" src="https://github.com/woocommerce/woocommerce/assets/186112/9b79ddd8-3f46-4d56-a568-25ac4935cede"> |



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Update the "Customer Account" block icon for the line style.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
